packages/dashboard-api/src/modules/networks/model.js

```diff
@@ -91,9 +106,61 @@ async function updateFiatVolume(
   return networks;
 }
 
+async function updateFiatVolumeV2(
+  networks = [],
+  tokensVolume24h = [],
+  tokensVolumeAllTime = [],
+  totalMintTokens = [],
+  totalBurnTokens = [],
+) {
+  let tokenNames = tokensVolume24h.concat(tokensVolumeAllTime).concat(totalMintTokens).concat(totalBurnTokens).map(item => item.tokenName);
+  tokenNames = [...new Set(tokenNames)];
+  const tokenPrices = await getPricesByTokenNames(tokenNames);
+  const tokenPriceProperties = setKeyValueTokenPrices(tokenPrices);
+  for (let networkInfo of networks) {
+    let USD24h = 0;
+    let USDAllTime = 0;
+    let USDMint = 0;
+    let USDBurn = 0;
+
+    for (const data of tokensVolume24h) {
+      if (data.networkId == networkInfo.id) {
+        const fiat = Number((Number(data.tokenVolume) * (tokenPriceProperties[data.tokenName] || 0)).toFixed(2));
+        USD24h += fiat;
+      }
+    }
+
+    for (const data of tokensVolumeAllTime) {
+      if (data.networkId == networkInfo.id) {
+        const fiat = Number((Number(data.tokenVolume) * (tokenPriceProperties[data.tokenName] || 0)).toFixed(2));
+        USDAllTime += fiat;
+      }
+    }
+
+    for (const data of totalMintTokens) {
+      if (data.networkId == networkInfo.id) {
+        const fiat = Number((Number(data.tokenValue) * (tokenPriceProperties[data.tokenName] || 0)).toFixed(2));
+        USDMint += fiat;
+      }
+    }
+
+    for (const data of totalBurnTokens) {
+      if (data.networkId == networkInfo.id) {
+        const fiat = Number((Number(data.tokenValue) * (tokenPriceProperties[data.tokenName] || 0)).toFixed(2));
+        USDBurn += fiat;
+      }
+    }
+
+    networkInfo.usd24h = numberToFixedAmount(USD24h);
+    networkInfo.usdAllTime = numberToFixedAmount(USDAllTime);
+    networkInfo.mintFee = USDMint;
+    networkInfo.burnFee = USDBurn;
+  }
+  return networks;
+}
 async function getNetworkById(networkId) {
   //	Get token_name from registered_tokens table related to networkId
-  const tokens = (await getTokensbyNetworkId(networkId)).map(element => element.token_name);
+  const tokens = (await getTokensbyNetworkId(networkId)).map((element) => element.token_name);
```

Comment:
Add `()` or not is the same

Reply:
it is default config when i use format shortcut